app/api/process.py

```diff
@@ -86,7 +86,11 @@
     'VT': [utils.standardize_data, close_outbreaks.close_outbreaks, utils.post_processing],
     'WI': [utils.standardize_data, close_outbreaks.close_outbreaks, utils.post_processing],
     'WV': [utils.standardize_data, utils.post_processing],
-    'WY': [utils.standardize_data, aggregate_outbreaks.collapse_outbreak_rows, utils.post_processing],
+    'WY': [utils.standardize_data,
+           aggregate_outbreaks.collapse_outbreak_rows,
+           close_outbreaks.close_outbreaks,
+           lambda df: utils.post_processing(df, close_unknown_outbreaks=True),
```

Comment:
You've looked at WY way more than I have, so feel free to ignore this comment if I'm not understanding the logic! That said, I'm a bit nervous about running any functions after post_processing - ie we're copy-pasting older info first and then rewriting the numbers, is that in the right order?

(Otherwise, if the main reason is to close unknown outbreaks, it might be safer to just make that its own tiny function run before `aggregate_outbreaks.sum_outbreaks` and keep running `post_processing` last)

Reply:
Yeah I didn't love putting it after post processing. The reason for that is because post_processing includes `fill_missing_dates` and we need to do this WY step after that, because we need the formerly missing closed outbreak rows. I'll change it to do `fill_missing_dates` and close the blank outbreak statuses as the first step of `sum_outbreaks`, and `post_processing` will just do them again too; neither takes more than a second.

Reply:
Done. I also added `State_Facility_Type` to the sort in post_processing because we found a facility that had both assisted living and nursing home where they were getting mixed up.